.goreleaser.yaml

```diff
@@ -0,0 +1,51 @@
+---
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    # Keep mod files clean
+    - go mod tidy -v
+
+builds:
+  - main: main.go
+    binary: bin/diff
+    env:
+      - CGO_ENABLED=0
+    flags: -trimpath
+    ldflags: -s -w -X {{ .ModulePath }}/cmd.Version={{ .Version }}
+    targets:
+      # helm supported targets
+      # https://github.com/helm/helm/blob/main/Makefile#L4C1-L4C1
+      - darwin_amd64
+      - darwin_arm64
+      - linux_386
+      - linux_amd64
+      - linux_arm
```

Comment:
If I'm not missing anything, we can safely omit `386`, `arm`, `ppc64le` and `s390x` "yet" as we don't publish binaries for those archs today.

Reply:
Just included all architectures supported by Helm. 
Yes, its safe to omit them, but why to not include it now?